templates/superset_config.py

```diff
@@ -50,9 +50,9 @@
     "force_https": False,
 }
 
-SQLALCHEMY_POOL_SIZE = 5  # Maximum number of connections in the pool
-SQLALCHEMY_POOL_TIMEOUT = 300  # Maximum time (in seconds) to wait for a connection
-SQLALCHEMY_MAX_OVERFLOW = 10
+SQLALCHEMY_POOL_SIZE = int(os.getenv('SQLALCHEMY_POOL_SIZE'))
```

Comment:
Does the operator framework enforce types of the environment variables from config? If not it might be better to be more explicit with the values you process, we do not want someone putting `foo` as the pool size.

Reply:
It does enforce - as part of the `config.yaml` you have `type: int`, if anything but an int is entered then the user gets an error message and the config value is not permitted.